config-example.yaml

```diff
@@ -0,0 +1,11 @@
+api_base_url: '/v1/students'
+user: 'username'
+password: 'password'
+multiple_courses: ['123456789']
```

Comment:
Can you give more of a description on what I should be looking for in osuIds? A config in env_files would help